api-participant/src/main/java/bio/terra/pearl/api/participant/service/AuthUtilService.java

```diff
@@ -38,11 +38,15 @@ public AuthUtilService(
    * returns the enrollee if the user is authorized to access/modify it, throws an error otherwise
    */
   public Enrollee authParticipantUserToEnrollee(UUID participantUserId, String enrolleeShortcode) {
-    // for now, a user is only allowed to access an enrollee if it's themself.  Later, we'll add
-    // proxies
+    // for now, a user is only allowed to access an enrollee if it's themselves or their proxy
```

Comment:
is this comment still relevant now that proxies have been added? is there another user-type that might be allowed access in the future?